### DISCUSSION TITLE
🟠 Nested Components | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#629] [Announcements]
# 🟠 Nested Components | Angular Challenges

Challenge 18 is about testing nested components

https://angular-challenges.vercel.app/challenges/testing/18-nested-comp/

**Comment by idembele70** [comment]
Don't worked for me: `npx nx component-test testing-nested`
But this worked for me: 
```sh
npx nx component-test testing-nested-components
```
